target/linux/ramips/dts/ApexeraWiBot.dts

```diff
@@ -0,0 +1,172 @@
+/dts-v1/;
```

Comment:
Please include licensing info for your DTS file, see
https://openwrt.org/submitting-patches#dts_checklist
e.g. a line like:
`// SPDX-License-Identifier: GPL-2.0-or-later OR MIT`
above the `/dts-v1/;`